bridgeifns.py

```diff
@@ -0,0 +1,194 @@
+# stdlib
+from os import name
+from typing import Tuple
+# lib
+from cloudcix.rcc import comms_ssh, CouldNotConnectException
+# local
+
+
+__all__ = [
+    'build',
+    'scrub',
+]
+
+SUCCESS_CODE = 0
+
+
+def build(
+    bridgename: str,
+    namespace: str,
+) -> Tuple[bool, str]:
+    """
+    description:
+        Creates a veth link on the main namespace and connects it to a bridge.
+        Then, it moves one end of the link to a VRF network namespace and sets the interface up.
+
+    parameters:
+        bridgename:
+            description: The name of the bridge on the main namespace.
+            type: string
+            required: true
+        namespace:
+            description: The VRF network namespace identifier, such as 'VRF123'.
+            type: string
+            required: true
+
+    return:
+        description: |
+            A tuple with a boolean flag indicating if the veth link creation was successful,
+            and the output or error message.
+        type: tuple
+    """
```

Comment:
I recommend structuring each primitive function in the following way

1. Define messages dictionary
2. Load config_file
3. Read `ipv6_subnet` from config_file and set podnet_a and podnet_b IPs
4. Read and compare the `podnet_an_enabled` and `podnet_b_enabled` from config file, one should be sure and the other false. This logic is used to determine the `enabled` and `disabled` podnet.
5. Define the payload requests request - https://github.com/CloudCIX/primitives/blob/87224b988c89702c81770ee3f32d91bf0b7cca3b/nginxns.py#L151
6. Send each payload request on enabled podnet via rcc
7. Send each payload request on disabled podnet via rcc
8. All the above was success, return True and success message from messages dictionary in step 1


For steps 6/7, if the channel_code or the payload_code in the response is not the expected value, an error has occurred and the channel/payload message and error appended to the message in the return statement e.g. - https://github.com/CloudCIX/primitives/blob/87224b988c89702c81770ee3f32d91bf0b7cca3b/directorymain.py#L119